src/irr/asset/CImageLoaderPNG.cpp

```diff
@@ -189,10 +193,19 @@ asset::IAsset* CImageLoaderPng::loadAsset(io::IReadFile* _file, const asset::IAs
 	else
```

Comment:
`screen_gamma` is actually 2.233333333

Reply:
Or try to set to 1.0 and then use the SRGB formats instead in `new asset::CImageData` later on

Reply:
actually always:1.0